Gemfile

```diff
@@ -43,10 +43,12 @@ gem 'travis',         '~> 1.7.6'
 
 gem 'awesome_spawn',        '>= 1.4.1'
 gem 'default_value_for',    '>= 3.1.0'
-gem 'haml_lint',            '~> 0.28.0', :require => false
+gem 'haml',                 '~> 5.1',    :require => false # force newer version of haml
```

Comment:
This is just to make this Rails 5.2 compatible.  Probably not "really" required, but decided to put it in since `haml_lint` has a very loose `haml` dependency.

Reply:
Well, if this gets pulled in as a dep into miq in development, this might be a problem. I'd feel safer with a `<5` here. cc @himdel

Reply:
We can't do that, for the exact reasons why you requested this be upgraded in the first place:  `haml < 5` is not Rails 5.2 compatible.

- https://github.com/haml/haml/issues/895#issuecomment-282551500
- https://github.com/jrafanie/manageiq/commit/0ae87ac93f041ad955914b4e640b2c42c2dc725f

So if you were to use `miq_bot` in a `.gemspec`, `haml > 5` would be required.

Reply:
I don't think we will include miq_bot in a gemspec, since it's not a gem, so I don't think there's an issue here.